WordPress/Classes/Services/NotificationActionsService.swift

```diff
@@ -16,7 +16,7 @@ class NotificationActionsService: LocalCoreDataService {
             return
         }
 
-        siteService.followSite(withID: siteID, site: nil) {
+        siteService.followSite(withID: siteID, topic: nil, post: nil) {
```

Comment:
The `topic` and `post` represent a post and/or topic in which the user is following this site?

Maybe a `fromTopic` or `fromPost` would be clear?